src/Services/TemplateService.php

```diff
@@ -137,6 +137,13 @@ public function renderTemplate($template, $params)
         return $renderedTemplate;
     }
 
+    public function isLegalPage(): bool {
+        /** @var ShopUrls $shopUrls */
+        $shopUrls = pluginApp(ShopUrls::class);
+
+        return $shopUrls->isLegalPage();
+    }
+
```

Comment:
Wir haben die ganzen anderen Funktionen in der Klasse bereits als `deprecated` markiert. Ich würde daher keine neue Funktion hinzufügen. Im Twig-Template müsstest du auch direkt auf `urls.isLegalPage()` zugreifen können...